SetupScript.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+if [ $EUID -ne 0 ]; then
+	echo "Please run as root user"
+	exit 1
+fi
+#base script to fresh install systems onto base image for raspberry pi
+
+#check internet connection
+ping -q -c1 google.com > /dev/null
+
+if [ $? -ne 0 ]; then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#configure displays
+git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git /opt/Epsilon-Raspberry/
+cp /opt/Epsilon-Raspberry/primary/xorg.conf /etc/X11/ 
+cp /opt/Epsilon-Raspberry/primary/config.txt /boot/
+
+#install rabbit-mq
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | tee /etc/apt/sources.list.d/rabbitmq.list && apt-get update && apt-get install rabbitmq-server
+apt-get install cmake \
+	libboost-dev \
+	openssl \
+	libssl-dev \
+	libblkid-dev \
+	e2fslibs-dev \
+	libboost-all-dev \
+	libaudit-dev \
+	software-properties-common \
+	build-essential \
+	mesa-common-dev \
+	libgl1-mesa-dev
+
+if [ -n "/usr/local/lib/librabbitmq" ]; then
+	echo "Rabbitmq already setup"
+else
+	git clone https://github.com/alanxz/rabbitmq-c /tmp/rabbitmq-c/
+	mkdir /tmp/rabbitmq-c/build
+	(cd /tmp/rabbitmq-c/build && cmake .. && cmake --build .)
+	cp /tmp/rabbitmq-c/build/librabbitmq/*.a /usr/local/lib/
+	cp /tmp/rabbitmq-c/build/librabbitmq/*.so* /usr/local/lib/
+fi
+
+#begin QT install
+apt-get update
+apt-get --yes upgrade
+apt-get install --yes -qq \
+	libfontconfig1-dev \
+	libdbus-1-dev \
+	libfreetype6-dev \
+	libudev-dev \
+	libicu-dev \
+	libsqlite3-dev \
+	libxslt1-dev \
+	libssl-dev \
+	libasound2-dev \
+	libavcodec-dev \
+	libavformat-dev \
+	libswscale-dev \
+	libgstreamer0.10-dev \
+	libgstreamer-plugins-base0.10-dev \
+	gstreamer-tools \
+	gstreamer0.10-plugins-good gstreamer0.10-plugins-bad \
```

Comment:
`gstreamer0.10-plugins-bad` isn't available anymore (at least on the pi), try replacing it with `gstreamer0.10-plugins-base` and `gstreamer0.10-plugins-ugly`